components/meta-packages/history/history

```diff
@@ -475,6 +475,7 @@ database/mariadb-55@5.5.55,5.11-2017.0.0.3
 database/mariadb-55/client@5.5.55,5.11-2017.0.0.3
 database/mariadb-55/library@5.5.55,5.11-2017.0.0.3
 database/mariadb-55/tests@5.5.55,5.11-2017.0.0.3
+database/mongodb-33@3.3.12-2018.0.0.3
```

Comment:
Current version is 3.3.12-2018.0.0.3. Make this package one version larger (3.3.12,5.11-2018.0.0.4)